pkg/trace/interpreter/util/topology.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+import (
+	"fmt"
+)
+
+// ServiceTypeName returns the default service type
+const ServiceTypeName = "service"
+
+// ServiceInstanceTypeName returns the default service instance type
+const ServiceInstanceTypeName = "service-instance"
+
+// CreateServiceURN creates the urn identifier for all service components
+func CreateServiceURN(serviceName string) string {
```

Comment:
I would avoid have `util` packages, can this be part of the `MakeDefaultSpanInterpreter` ?

Reply:
Moved this into the interpreter interface

Reply:
cool